config/openiotsdk/cmake/sdk.cmake

```diff
@@ -88,6 +88,9 @@ endif()
 if(TFM_PROJECT_CONFIG_HEADER_FILE)
     set(TFM_CMAKE_ARGS "${TFM_CMAKE_ARGS};-DPROJECT_CONFIG_HEADER_FILE=${TFM_PROJECT_CONFIG_HEADER_FILE}")
 endif()
+if ("${CONFIG_CHIP_CRYPTO}" STREQUAL "psa")
+    set(TFM_CMAKE_ARGS "${TFM_CMAKE_ARGS};-DMBEDCRYPTO_PATH=${CMAKE_BINARY_DIR}/_deps/mbedtls-src")
```

Comment:
I don't think it will point to the right path every time. For example if mbedtls is provided externally using https://cmake.org/cmake/help/latest/module/FetchContent.html#variable:FETCHCONTENT_SOURCE_DIR_%3CuppercaseName%3E . 

I'd expect it to be retrieved with `FetchContent_GetProperties(mbedtls)` and then using the variable `${mbedtls_SOURCE_DIR}`. However we are setting this variable before mbedtls is populated 🙃 .

In that case I think we can look if `FETCHCONTENT_SOURCE_DIR_MBEDTLS` has been set. 
If it has been set, use its value as the mbed crypto path otherwise use `${FETCHCONTENT_BASE_DIR}/mbedtls-src`